src/Support/SupportData.php

```diff
@@ -0,0 +1,930 @@
+		$slug = $slug[0];
+
+		$amp_options        = get_option( 'amp-options' );
+		$suppressed_plugins = ( ! empty( $amp_options['suppressed_plugins'] ) && is_array( $amp_options['suppressed_plugins'] ) ) ? $amp_options['suppressed_plugins'] : [];
+
+		$suppressed_plugin_list = array_keys( $suppressed_plugins );
+
+		if ( empty( $plugin_data['Name'] ) ) {
+			return [];
+		}
+
+		return [
+			'name'              => $plugin_data['Name'],
+			'slug'              => $slug,
+			'plugin_url'        => array_key_exists( 'PluginURI', $plugin_data ) ? $plugin_data['PluginURI'] : '',
+			'version'           => array_key_exists( 'Version', $plugin_data ) ? $plugin_data['Version'] : '',
+			'author'            => array_key_exists( 'AuthorName', $plugin_data ) ? $plugin_data['AuthorName'] : '',
+			'author_url'        => array_key_exists( 'AuthorURI', $plugin_data ) ? $plugin_data['AuthorURI'] : '',
+			'requires_wp'       => array_key_exists( 'RequiresWP', $plugin_data ) ? $plugin_data['RequiresWP'] : '',
+			'requires_php'      => array_key_exists( 'RequiresPHP', $plugin_data ) ? $plugin_data['RequiresPHP'] : '',
+			'is_active'         => is_plugin_active( $plugin_file ),
+			'is_network_active' => is_plugin_active_for_network( $plugin_file ),
+			'is_suppressed'     => in_array( $slug, $suppressed_plugin_list, true ) ? $suppressed_plugins[ $slug ]['last_version'] : '',
+		];
+
+	}
+
+	/**
+	 * To normalize theme information.
+	 *
+	 * @since 2.2
+	 *
+	 * @param \WP_Theme $theme_object Theme object.
+	 *
+	 * @return array Normalize theme information.
+	 */
+	public static function normalize_theme_info( $theme_object ) {
+
+		if ( empty( $theme_object ) || ! is_a( $theme_object, 'WP_Theme' ) ) {
+			return [];
+		}
+
+		$active_theme      = wp_get_theme();
+		$active_theme_slug = '';
+		$parent_theme      = '';
+
+		if ( ! empty( $active_theme ) && is_a( $active_theme, 'WP_Theme' ) ) {
+			$active_theme_slug = $active_theme->get_stylesheet();
+		}
+
+		if ( ! empty( $theme_object->parent() ) && ! is_a( $theme_object->parent(), 'WP_Theme' ) ) {
+			$parent_theme = $theme_object->parent()->get_stylesheet();
+		}
+
+		$tags = $theme_object->get( 'Tags' );
+		$tags = ( ! empty( $tags ) && is_array( $tags ) ) ? $tags : [];
+
+		$theme_data = [
+			'name'         => $theme_object->get( 'Name' ),
+			'slug'         => $theme_object->get_stylesheet(),
+			'version'      => $theme_object->get( 'Version' ),
+			'status'       => $theme_object->get( 'Status' ),
+			'tags'         => $tags,
+			'text_domain'  => $theme_object->get( 'TextDomain' ),
+			'requires_wp'  => $theme_object->get( 'RequiresWP' ),
+			'requires_php' => $theme_object->get( 'RequiresPHP' ),
+			'theme_url'    => $theme_object->get( 'ThemeURI' ),
+			'author'       => $theme_object->get( 'Author' ),
+			'author_url'   => $theme_object->get( 'AuthorURI' ),
+			'is_active'    => ( $theme_object->get_stylesheet() === $active_theme_slug ),
+			'parent_theme' => $parent_theme,
```

Comment:
Or rather than this can just be:

```suggestion
			'parent_theme' => $theme_object->get_template(),
```